src/encoded/static/components/matrix_mouse_development.js

```diff
@@ -351,12 +289,9 @@ const convertExperimentToDataTable = (context, getRowCategories, mapRowCategoryQ
     if (stageFilter) {
         stageFilter.forEach((f) => {
             if (['adult', 'postnatal', 'embryo'].includes(f)) {
-                selectedFilters += `replicates.library.biosample.life_stage=${f === 'embroyo' ? 'embryonic' : f}&`;
+                selectedFilters += `replicates.library.biosample.life_stage=${f === 'embryo' ? 'embryonic' : f}&`;
```

Comment:
Well, stageFilter is the array of filters, and this is basically where we need to do the .join(). But it's a little more complicated than a join because in some cases we want it joined by `replicates.library.biosample.life_stage=` and in other cases by `life_stage_age=`, and we have to replace "embryo" by "embryonic".